releasenotes/notes/ad-fix-annotation-value-parsing-152e1a2b2fe10891.yaml

```diff
@@ -0,0 +1,8 @@
+---
+fixes:
+  - |
+    Fix an auto-discovery annotation value parsing limitation in version 6
+    compare to version 5.
+    Now, `ad.datadoghq.com/*.instances` annotation key supports value like `[[{"foo":"bar1"}, {"foo":"bar2"}], {"name":"bar3"}]`
```

Comment:
the format is RST so you want double backquote for `monospaced text`

see http://docutils.sourceforge.net/docs/user/rst/quickref.html#inline-markup

nit: could you remove the extra lines as well?

Reply:
thanks for the link, commit updated with fixes